src/pages/home/home.scss

```diff
@@ -1,3 +1,19 @@
-page-home {
-
+.page-home {
+  background-color: #DCEBFD;
+  .header {
+    font-size: 18pt;
+    margin-top: 10pt;
+    margin-bottom: 10pt;
+    padding: 0 10pt;
+    img {
+      height: 28pt;
+      width: 28pt;
+    }
+  }
+  ion-item{
+    border-color: white;
+    &.stripped{
+      background: repeating-linear-gradient(45deg, #F6FAFC, #F6FAFC 15px, white 15px, white 30px);
```

Comment:
Expected "#F6FAFC" to be "#f6fafc"    color-hex-case<br>Unexpected named color "white"        color-named